cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2490,6 +2490,9 @@ private boolean isPodToRestart(StatefulSet ss, Pod pod, boolean isAncillaryCmCha
                     if (ca.certsRemoved()) {
                         reasons.add(ca + " certificate removal");
                     }
+                    if (ca.certInfoChanged()) {
+                        reasons.add(ca + "certificate info change");
```

Comment:
I would probably use somethinf else than `info`. I do not think anyone knows what does `certificate info` mean.